python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -207,9 +208,10 @@ def _make_sensor(
     name: str,
     wait_for_all_upstream: bool,
     wait_for_in_progress_runs: bool,
-    minimum_interval_seconds: Optional[int] = None,
-    description: Optional[str] = None,
-    default_status: DefaultSensorStatus = DefaultSensorStatus.STOPPED,
+    minimum_interval_seconds: Optional[int],
```

Comment:
I forget if our best practice is to not have these params with defaults since they are passed through from `build_asset_reconciliation_sensor`. can add them back if they should be there